django/db/migrations/questioner.py

```diff
@@ -214,6 +219,30 @@ def ask_not_null_alteration(self, field_name, model_name):
                 return self._ask_default()
         return None
 
+    def ask_not_null_removal(self, field_name, model_name):
+        """Removing a NOT NULL field from a model"""
+        if not self.dry_run:
+            choice = self._choice_input(
+                f"You are trying to remove a non-nullable field '{field_name}' from "
+                f"'{model_name}' without a default; "
+                f"we can't do that (the database needs something to populate existing "
+                f"rows when migrating backwards).\n"
```

Comment:
Please void personalised language. Take a look at the add not-null-no-default-field question for guidance.

Also "can't do that" is misleading. Some people may be quite happy to proceed without a valid reverse operation.